packages/@aws-cdk/aws-eks/test/test.cluster.ts

```diff
@@ -1162,6 +1162,26 @@ export = {
       test.done();
     },
 
+    'addNodegroup with T4g instance type comes with nodegroup with correct AmiType'(test: Test) {
```

Comment:
Also add a test for the auto selection of the cpu arch when creating an image for auto-scaling group capacity:

https://github.com/aws/aws-cdk/blob/028495e55fed02f727024a443fc29a17d4629fe3/packages/%40aws-cdk/aws-eks/lib/cluster.ts#L1795-L1799

Reply:
DONE